File: resources/js/acc.js
$(document).ready(function () {
	$.getJSON('resources/php/acc.py')
		.done(function (data) {
			$('#acc-userid').val(data.userId || '');
			$('#acc-username').val(data.username || '');
			$('#acc-email').val(data.email || '');
		});
});

File: resources/php/acc.py
from flask import jsonify, session


def register_routes(app):
    @app.route("/resources/php/acc.py", methods=["GET", "POST"])
    def acc_php():
        return jsonify(
            {
                "userId": session.get("userId", ""),
                "username": session.get("username", ""),
                "email": session.get("email", ""),
            }
        )

